CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+
+Before creating enhancement suggestions, please check issues as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please include as many details as possible. Fill in the template, including the steps that you imagine you would take if the feature you're requesting existed.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on that repository and provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps or point out the part of code which the suggestion is related. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://gitlab.gnome.org/Archive/byzanz) on Linux.
+* **Explain why this enhancement would be useful** to most Terraform users and isn't something that can or should be implemented as a community package.
+* **Specify which version of Terraform you're using.** You can get the exact version by running `terraform -v` in your terminal.
+* **Specify the name and version of the OS you're using.**
+
+
+## Pull Request Process
+
+The process described here has several goals:
+
+- Fix problems that are important to users.
+- Engage the community in working toward the best possible terraform module.
+- Enable a sustainable system for DNXers maintainers to review contributions.
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+5. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+6. Follow the [styleguides](#styleguides)
```

Comment:
include Terraform styleguides.
You can take from this one https://github.com/jonbrouse/terraform-style-guide/blob/master/README.md
To not make this file to big, create a new file with the styleguides inside the repo and link it here.

Reply:
Done ✅